client/src/pages/locations/GeoLocation.js

```diff
@@ -0,0 +1,36 @@
+import React from "react"
+import PropTypes from "prop-types"
+import { Button } from "react-bootstrap"
+import { Location } from "../../models"
+import REMOVE_ICON from "../../resources/delete.png"
+import { Coordinate } from "./Show"
+
+const GeoLocation = ({ lat, lng, onClearLocation, isSubmitting }) => {
+  return (
+    <>
+      <Coordinate coord={lat} />,&nbsp;
+      <Coordinate coord={lng} />
+      {
+        Location.hasCoordinates({ lat, lng }) && typeof onClearLocation === "function" &&
```

Comment:
The prop.type guarantees that `onClearLocation` is a function; do you perhaps mean to test whether it has been assigned value or not?